### ISSUE TITLE
[error] PHP Notice:  Undefined property: DOMText::$tagName

### ISSUE THREAD

**Issue by laplasz** [mkweb/pupnp#4]
[Sun May 11 22:21:19 2014] [error] PHP Notice:  Undefined property: DOMText::$tagName in /var/www/pupnp/src/at/mkweb/upnp/backend/xmlparser/ServiceXMLParser.php on line 271
[Sun May 11 22:21:19 2014] [error] PHP Notice:  Undefined property: DOMText::$tagName in /var/www/pupnp/src/at/mkweb/upnp/backend/xmlparser/ServiceXMLParser.php on line 264
[Sun May 11 22:21:19 2014] [error] PHP Notice:  Undefined property: DOMText::$tagName in /var/www/pupnp/src/at/mkweb/upnp/backend/xmlparser/ServiceXMLParser.php on line 267
[Sun May 11 22:21:19 2014] [error] PHP Notice:  Undefined property: DOMText::$tagName in /var/www/pupnp/src/at/mkweb/upnp/backend/xmlparser/ServiceXMLParser.php on line 228

Maybe my router causes this problem:

2014-05-11 22:21:19 - 192.168.100.102 - [DEBUG] at\mkweb\upnp\backend\Client::__construct; Device: uuid:upnp-InternetGatewayDevice-192168100178900001 [TP-LINK Wireless Router WR340G]; Service: ConnectionManager

There is no Sink or Source at all...

**Comment by laplasz** [comment]
yes, that was the problem - when I disabled the router UPnP feature I finally got a device list in the front end.

**Comment by mkweb** [comment]
Same issue as in #3, it does not check if the device has basic media functions and just requests it. Will get fixed soon.
